en/scripting/modules/engine.md

```diff
@@ -0,0 +1,85 @@
+# Modules
+
+The engine and editor expose functional interfaces to developers via modules, which exist as **ECMAScript** modules.
+
+> **Note**: engine functions will not be accessible via global variables prefixed with `cc`!
+
+## Engine Modules
+
+### Function
+
+The module `'cc'` provides access to all engine functions. The contents of module `'cc'` are dynamic and are related to the **Feature Crop** setting in **Project Setting**.
```

Comment:
functions -> functionalities?

Reply:
functions is ok.

if you use:  functionalities/features then pluralize engine too.

This: The module `'cc'` provides access to all engine functions.
or: The module `'cc'` provides access to all engine functionality.
or: The module `'cc'` provides access to all of the engine's functionality.
or: The module `'cc'` provides access to all of the engine's features.